src/ui/public/vis/editors/default/default.html

```diff
@@ -1,5 +1,15 @@
-<div class="collapsible-sidebar">
+<div class="collapsible-sidebar" ng-class="{
+    'collapsible-sidebar-small': vis.type.editorConfig.defaultSize === 'small',
+    'collapsible-sidebar-medium': vis.type.editorConfig.defaultSize === 'medium',
+    'collapsible-sidebar-large': vis.type.editorConfig.defaultSize === 'large'
+  }">
   <vis-editor-sidebar class="vis-editor-sidebar" />
+  <vis-editor-resizer
+    id="editor_resizer"
```

Comment:
According to our HTML styleguide, our ids should be camelCased (`editorResizer`).